### DISCUSSION TITLE
quarto preview not responding to updating an "included" file

### DISCUSSION THREAD

**Discussion by kris-brown** [quarto-dev/quarto-cli#2969] [Q&A]
Hi, I use the `include` feature to share content among multiple `.qmd` files. However, when I update the content, `quarto preview` does not know that it should re-render the `.qmd` file that has included the changed file. Is there any workaround for this?

**Comment by cscheid** [comment]
Unfortunately, there currently isn't. It's a known issue but it'll be a while (6+ months) until we're in a place where we can appropriately fix it (because the feature interacts with a lot of other things in quarto.)
